_includes/cloudcode/cloud-code-advanced.md

```diff
@@ -583,6 +583,18 @@ Here's an example of the JSON data that would be sent in the request to this web
 
 After setting up your webhook in the Dashboard UI, you'll be acurately decrementing comment counts!
 
+# Sending Emails
+If your Parse Server configuration specifies an `emailAdapter`, you can send mail via cloud code using `Parse.Cloud.sendMail`.
+
+```javascript
+Parse.Cloud.sendMail({
+  from: 'Example <test@example.com>',
+  to: 'contact@example.com',
+  subject: 'Test email.',
```

Comment:
Totally trivial, but I think the dot should be in the `text`, not in the `subject`